docs/first-order/receptive_fields.ipynb

```diff
@@ -23,7 +23,7 @@
   },
```

Comment:
<p>probably good to reword. I think this was gotten from josh siegle</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>